pkg/logql/expr.y

```diff
@@ -133,19 +135,31 @@ logExpr:
     ;
 
 logRangeExpr:
```

Comment:
I don't love how we're duplicating all paths here. I know we've done this in the path, but it's getting out of hand (not your fault).

@cyriltovena we should consider refactoring this a bit, but I'm fine with this PR.